### ISSUE TITLE
Accessibility: Reader Filters

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#10734]
###### Accessibility Scanner Aduit 

<img src="https://user-images.githubusercontent.com/1509205/68092013-8f2b0e80-fe3b-11e9-9769-50c0d360faeb.png" alt="Screenshot_20191030-231348" width="400" />

* Filter `org.wordpress.android:id/filter_spinner` - Touch target

	This item's height is 47dp. Consider making the height of this touch target 48dp or larger.

* Follow Button org.wordpress.android:id/follow_button - Touch target

	This item's height is 30dp. Consider making the height of this touch target 48dp or larger.

* Nav Bar Icons  `org.wordpress.android:id/nav_icon` (navbar_post_item) - Image contrast

	The image's contrast ratio is 2.33. This ratio is based on an estimated foreground color of `#FFFFFF` and an estimated background color of `#A7AAAD`. Consider increasing this ratio to 3.00 or greater.

* Discover's Domain - `org.wordpress.android:id/text_domain` - Text contrast

	The item's text contrast ratio is 4.20. This ratio is based on an estimated foreground color of #787C82 and an estimated background color of `#FFFFFF`. Consider using a contrast ratio greater than 4.50 for small text, or 3.00 for large text.

* Follow Count `org.wordpress.android:id/text_blog_follow_count` - Text contrast

	The item's text contrast ratio is 3.91. This ratio is based on an estimated foreground color of `#787C82` and an estimated background color of `#F6F7F7.` Consider using a contrast ratio greater than 4.50 for small text, or 3.00 for large text.

* Discover Text `org.wordpress.android:id/text_discover` - Text contrast

	The item's text contrast ratio is 3.16. This ratio is based on an estimated foreground color of #8E9196 and an estimated background color of #FFFFFF. Consider using a contrast ratio greater than 4.50 for small text, or 3.00 for large text.

* Discover List Item's Visit Layout - `org.wordpress.android:id/layout_discover` - Touch target

	This item's height is 32dp. Consider making the height of this touch target 48dp or larger. A parent container may be handling touch events for this item. If selecting the larger container performs the same action as selecting this item, consider defining this item as not clickable. If a different action is performed, consider increasing the size of this item.

* Visit Button `org.wordpress.android:id/visit`- Touch target

	This item's height is 36dp. Consider making the height of this touch target 48dp or larger. A parent container may be handling touch events for this item. If selecting the larger container performs the same action as selecting this item, consider defining this item as not clickable. If a different action is performed, consider increasing the size of this item.

* More Button `org.wordpress.android:id/image_more` - Touch target

	This item's size is 36dp x 36dp. Consider making this touch target 48dp wide and 48dp high or larger. A parent container may be handling touch events for this item. If selecting the larger container performs the same action as selecting this item, consider defining this item as not clickable. If a different action is performed, consider increasing the size of this item.

##### Talkback Audit
<img src="https://user-images.githubusercontent.com/1509205/68091986-53904480-fe3b-11e9-9297-055067676852.png" alt="Screenshot_20191030-224532" width="400" />

* The words "site icon" are being read out loud when the user however over the header of a post in the Reader view. The post title and other pertinent information can be made available but it's okay to disregard the author's profile image since their name is being announced.

* Empty states should be read aloud. If enter has to be pressed that should also be expressed. 

* Saved posts : 
<img src="https://user-images.githubusercontent.com/1509205/68092022-ac5fdd00-fe3b-11e9-86d4-b70f9756653e.png" alt="Screenshot_20191101-201554"  width="400"/>

* Empty state should be announced. 

Tap to save post to your list. Icon shouldn't be read because the user wont know what it means anyway. 

* Discover : 
<img src="https://user-images.githubusercontent.com/1509205/68092044-f0eb7880-fe3b-11e9-8f39-d213c1b2109b.png" alt="Screenshot_20191101-202039" width="400" />

Summary of discovery read out and then the number of followers read in the same line. There should be some form of break in this. 
* Preview :  no issues noticed. 
* Post View : 
No issues noticed.
